app/assets/stylesheets/shared/shared.scss

```diff
@@ -22,3 +22,4 @@
 @import "table";
 @import "tabs";
 @import "toast";
+@import "calendar";
```

Comment:
for some reason, adding a `.scss` file to the `/app` directory wasn't properly being read, so i've added it here in the `/shared` directory for now. could also be a future discussion for if we want to have shared calendar styling across apps.

Reply:
Hmm... so you added the file `app/assets/stylesheets/app/calendar.scss`, then add an `@import "calendar"` to the `app/index.scss` right?

I would have expected that to work - things should generally work now that i've added explicit imports of the "barrel files" (`@import "app/index"` instead of just `@import "app"`).

Reply:
i will `merge main` and try again, but i did try `@import "calendar"` in the `index` file and added the `calendar` file and it didn't seem to work. i even tried to follow the audio segmenter patterning and make a new directory but it didn't seem to work either.

Reply:
I think [this is the commit](https://github.com/PRX/feeder.prx.org/commit/4934663958af36073b7bdbd37b26c09b3bb41d98) that fixed it.

Doing an `@import "audio-segmenter"` was bad at getting any updated files.  It needed the `/index` on it.